client/src/components/RangeSlider/RangeSlider.js

```diff
@@ -0,0 +1,22 @@
+import 'react-input-range/lib/css/index.css';
+import './RangeSlider.css';
+
+import React, { useState } from 'react';
+
+import InputRange from 'react-input-range';
+
+const RangeSlider = () => {
+  const [value, setValue] = useState({ min: 20000, max: 80000 });
```

Comment:
Comments for why _these_ magic numbers would be nice, and what they are supposed to reflect (guessing impressions?)